src/wrappers/sqs-consumer.js

```diff
@@ -153,9 +159,9 @@ function sqsConsumerWrapper(wrappedFunction) {
         utils.debugLog('sqs-consumer - inside wrapper');
         utils.debugLog(`sqs-consumer - options: ${options}`);
         const app = wrappedFunction.apply(this, [options]);
-        const patchedCallback = message => traceContext.RunInContext(
+        const patchedCallback = (message, consumer) => traceContext.RunInContext(
             tracer.createTracer,
-            () => sqsConsumerMiddleware(message, app)
+            () => sqsConsumerMiddleware(message, consumer, app)
```

Comment:
is the consumer  argument is mandatory?

Reply:
yes, it will be undefined if it's not sent